x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -204,6 +221,19 @@ export const getManagementFilteredLinks = async (
     (canReadHostIsolationExceptions &&
       (await checkArtifactHasData(HostIsolationExceptionsApiClient.getInstance(core.http))));
 
+  if (ExperimentalFeaturesService.get()?.riskScoringRoutesEnabled) {
```

Comment:
I see the `SecurityPageName.entityAnalyticsManagement` LinkItem definition already has the `experimentalKey: 'riskScoringRoutesEnabled'` defined, this properly will automatically filter the link if the experimental key is not enabled. 
There's no need to conditionally add a category, `categories` should be static, if there's no link enabled in the category it will not be displayed.